### ISSUE TITLE
(firefox, firefoxesr, thunderbird) Ignoring user / password of proxy while fetching firefox installer beginning 92.0.1

### ISSUE THREAD

**Issue by Jos3pherus** [chocolatey-community/chocolatey-packages#1727]
## Expected Behavior
Installations script downloads the firefox installer from mozilla.org and does the installation. This is done in our case via proxy with authentication (user / password). The credentials and proxy information are used from the chocolatey configuration.

## Current Behavior
While trying to download the installer from mozilla.org the proxy is used but not with credentials. So the error message is always that the download fails due to being not authorized to download the package.

## Possible Solution
Use the proxy user and password from chocolatey like it was done in Version 92.0 and lower. When we install Version 92.0 everything works without issues.

For me it looks like the issue is within a change in the helpers.ps 

## Steps to Reproduce (for bugs)
1. Configure a Proxy with user and password
2. Try to install firefox

## Your Environment
* Package Version used: 93.0 and 92.0.1
* Operating System and version: Windows 10
* Chocolatey version: latest version

**Comment by Jos3pherus** [comment]
> 
> 
> @Jos3pherus is it the actual download of the installer that is not going through a proxy?
> 
> Or is that happening before downloading the actual installer?
> 
> If it is the latter, then it is most likely related to recently addition of acquiring a correct fallback culture value and is covered in #1729

Hi,

in the log you can see that the installation scripts are downloaded from chocolatey correctly. When he runs the scripts he tries to download the installation .exe from mozilla.org which fails. You can see in the logs that he is using the configured proxy but without credentials. So he gets an Error 407 back from proxy (also visible in proxy log) that credentials are missing.
For testing purpose we whitelisted the IP of the machine so that it can use the proxy without credentials and installation worked fine. So it is just the missing credentials which are not forwarded to the webrequest (in my opinion).

So in short: The download of the scripts works fine, but the download of the executable from mozilla.org doesn't use the proxy credentials.

**Comment by AdmiringWorm** [comment]
@Jos3pherus just to be certain again, whether we are talking about the same thing or not.

The URL that you needed to whitelist, was this a URL going to `https://mozilla.org`, or to `https://download.mozilla.org`?

The latter one is currently expected to go through a proxy correctly, but the former is not guaranteed at the moment to go through a proxy, or may only partially go through a proxy.

**Comment by pauby** [comment]
This issue will be closed by PR #1732, which hasn't been merged yet. Reopening the issue until that time.

**Comment by drdamour** [comment]
How quickly does a fix turn into an available package?  Ive never experienced this before
